src/ws-dropdown/ws-dropdown.scss

```diff
@@ -0,0 +1,255 @@
+$export-dropdown-css: true !default;
```

Comment:
I think it makes sense to release this to fabric-design/scss

Reply:
Actually I moved it from scss to this place as the dropdown is only used together with the component and I guess no one will ever implement it's own logic of it.

Reply:
Funny fact: We had to do it in Retail Portal as it was easier for the team to implement dropdown on it's own then integrating wsDropdown (which is mainly because they had concerns after not knowing how to easily start...). 
I'm still for separation, as there are even 1-2 teams outside retail core who are only using SCSS library, even if we never thought about this before, people start using scss more than the components (pattern lib) we have.

Reply:
I'm fine with moving all the css to fabric-scss when we all align on it :)